.github/workflows/main.yml

```diff
@@ -155,7 +158,6 @@ jobs:
         if [[ ${WITHOUT_VCS} ]]
         then
           rm -rf .git
-          ${MX_PATH}/mx --primary-suite-path substratevm native-image --help
-        else
-          ${MX_PATH}/mx --primary-suite-path ${PRIMARY} --J @"-Xmx2g" --java-home=${JAVA_HOME} gate --strict-mode --tags ${GATE}
         fi
+
+        ${MX_PATH}/mx --primary-suite-path ${PRIMARY} --J @"-Xmx2g" --java-home=${JAVA_HOME} gate --strict-mode --tags ${GATE}
```

Comment:
This will build a GraalVM release that contains the `/substratevm` suite.
After running this
`$(mx --primary-suite-path ${PRIMARY} graalvm-home)` contains a working `bin/native-image` bash-launcher.
i.e. this looks good.

Reply:
Unfortunately this is not enough to test if things work outside a VCS repo, please see https://github.com/oracle/graal/pull/3478#issuecomment-861811175 for more details.

Reply:
Since https://github.com/oracle/graal/commit/6eee91e4e2229e53ced239617e504c519723ab8d got merged almost all substrate-related mx commands (like e.g. `mx native-image`) are just calling artifacts that got built as part of running `mx build` in the substratevm directory. In other words if `mx build` (or `mx gate --tags build`) succeeded you can safely assume that `mx native-image --help` also works since its just calling the native-image bash launcher that is located at `$(mx --primary-suite-path ${PRIMARY} graalvm-home)/bin/native-image` after `mx build` made sure it gets created there.

Reply:
I guess we could also do this:
```
PRIMARY=vm
DYNAMIC_IMPORTS=/substratevm,/sulong,/tools
NATIVE_IMAGES=polyglot # to save some time
GATE=build
```

It's not literally building everything, but it would exercise almost every different category of stuff there is to build (e.g. Java projects, native projects, a native-image launcher, ...).

Reply:
I see, I admit I didn't check the latest state of `mx_substratevm.py`, my mistake. Thanks for making this clear Paul.

Reply:
> It's not literally building everything, but it would exercise almost every different category of stuff there is to build (e.g. Java projects, native projects, a native-image launcher, ...).

Yup that would give pretty good coverage.

Reply:
Done, and seems to be working (at least on github, still waiting for the internal gate to finish).